app/services/dashboard/latest_manageable_cohort.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Dashboard
+  class LatestManageableCohort < BaseService
+    attr_reader :school
+
+    def initialize(school)
+      @school = school
+    end
+
+    def call
+      [
+        Cohort.latest,
+        pilot? ? Cohort.active_registration_cohort : Cohort.current,
+      ].compact.min_by(&:start_year)
```

Comment:
I might be missing something but I don't know if this could not be summed up as:
```ruby
pilot?  ? Cohort.active_registration_cohort : Cohort.current
```
Won't that always be the case?  The `Cohort.latest` worries me a bit for what it will do in test scenarios or if things are setup ahead of time in the future, it could get misused and it's only needed for this method?

Reply:
Bear in mind that this is computing a min. 
The `Cohort.latest` thing is just useful in cases (specs) where the current cohort has not been created yet, therefore, returning the most recent cohort created rather than returning nil and breaking the test.
In prod, we will always have the Cohort.current created. If not, it would return the previous.

Reply:
`Cohort.current` should only change when a new cohort is added with a `registration_start_date` and an `academic_year_start_date` that takes over from the previous one?

Reply:
`Cohort.current` is only based on the current date.

Reply:
Thanks for clarifying @ltello, I still don't think this should be here especially if it's just a "hack" to fix specs.  My main concern is `Cohort.latest` as I don't think that should exist because I think it could too easily be misunderstood and we have the necessary methods we need in `current`,`previous` and `next` on `Cohort`.  I don't want to hold up this PR though but can you add a tech debt ticket to get rid of this and correct the specs that might be missing a Cohort creation and/or `travel_to` blocks to work correctly with the cohorts used in those specs?